azure/services/asogroups/spec.go

```diff
@@ -55,6 +55,9 @@ func (s *GroupSpec) Parameters(ctx context.Context, object genruntime.MetaObject
 	return &asoresourcesv1.ResourceGroup{
 		ObjectMeta: metav1.ObjectMeta{
 			OwnerReferences: []metav1.OwnerReference{s.Owner},
+			// Annotations:     map[string]string{
+			//   "serviceoperator.azure.com/credential-from": "my-resource-secret",
+			// },
```

Comment:
why was this commented? can it be removed?

Reply:
I'll remove it once the PR is ready to be merged. I left it here as an example to discuss around how do we want to pass the ASO `credential-from` label